tests/src/test/java/me/aurium/beetle/branch/tests/AbstractTests.java

```diff
@@ -0,0 +1,26 @@
+package me.aurium.beetle.branch.tests;
+
+import me.aurium.beetle.branch.adapter.ContextAdapterFactory;
+import me.aurium.beetle.branch.adapter.DelegatingContextAdapterFactory;
+import me.aurium.beetle.branch.tests.mockups.MockupContextSource;
+import org.junit.platform.commons.logging.Logger;
+import org.junit.platform.commons.logging.LoggerFactory;
+
+public abstract class AbstractTests {
+
+    protected final Logger logger = LoggerFactory.getLogger(BranchingNodeTests.class);
+
+    protected final ContextAdapterFactory<String> adapter = new DelegatingContextAdapterFactory<>(new MockupContextSource(logger));
+
+    protected final static String SENDER = "Tester";
+    protected final static String ALIAS = "TestCommand";
+
+    protected String[] of(String... strings) {
+        return strings;
```

Comment:
Why?

Reply:
Laziness

Reply:
Prioritize the time of the reader over the time of the writer. If I saw `of(...)` rather than `new String[] {...}` I would have to first visit the of method and see what it does

Reply:
_sigh_ you're right, i'll set it to an array.

Reply:
Resolved